airbyte-cdk/java/airbyte-cdk/db-sources/src/main/java/io/airbyte/cdk/integrations/debezium/internals/mssql/MssqlCdcStateConstants.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.integrations.debezium.internals.mssql;
+
+public class MssqlCdcStateConstants {
+
+  public static final String MSSQL_CDC_OFFSET = "mssql_cdc_offset";
+  public static final String MSSQL_DB_HISTORY = "mssql_db_history";
+  public static final String IS_COMPRESSED = "is_compressed";
+  public static final boolean COMPRESSION_ENABLED = true;
+
+}
```

Comment:
Considering that these constants all get referenced in just one class, why not define them there?